feature/gribi/mpls_in_udp/otg_tests/mpls_in_udp_scale/mpls_in_udp_scale_test.go

```diff
@@ -0,0 +1,1470 @@
+	case mplsPacketCount == 0:
+		return fmt.Errorf("no UDP-IPv6 packets found on port %s", otgPortName)
+	case validMplsPacketCount == 0:
+		return fmt.Errorf("no valid MPLS-in-UDP packets found on port %s", otgPortName)
+	case validMplsPacketCount < mplsPacketCount/2:
+		return fmt.Errorf("too many invalid packets: %d/%d",
+			mplsPacketCount-validMplsPacketCount, mplsPacketCount)
+	}
+
+	t.Logf("Validation PASSED: %d valid MPLS-in-UDP packets", validMplsPacketCount)
+	return nil
+}
+
+// Main test entry point.
+func TestMPLSinUDPScale(t *testing.T) {
+	t.Helper()
+	dut := ondatra.DUT(t, "dut")
+	ate := ondatra.ATE(t, "ate")
+	ctx := context.Background()
+	vrfsList := configureDUT(t, dut)
+	ateConfig := configureOTG(t, ate, dut)
+	ate.OTG().PushConfig(t, ateConfig)
+	ate.OTG().StartProtocols(t)
+	otgutils.WaitForARP(t, ate.OTG(), ateConfig, "IPv6")
+
+	// Configure gRIBI client
+	c := gribi.Client{
+		DUT:         dut,
+		FIBACK:      true,
+		Persistence: true,
+	}
+
+	if err := c.Start(t); err != nil {
+		t.Fatalf("gRIBI Connection can not be established")
+	}
+	defer c.Close(t)
+	c.BecomeLeader(t)
+
+	// Flush all existing AFT entries and set up basic routing infrastructure
+	c.FlushAll(t)
+	programBasicEntries(t, dut, &c)
+	t.Run("Profile-1-Single VRF", func(t *testing.T) {
+		if err := configureVrfProfiles(ctx, t, ateConfig, dut, ate, &c, []string{deviations.DefaultNetworkInstance(dut)}, 1, false); err != nil {
+			t.Errorf("configureVrfProfiles failed: %v", err)
+		}
+	})
+	t.Run("Profile-2-Multi VRF", func(t *testing.T) {
+		if err := configureVrfProfiles(ctx, t, ateConfig, dut, ate, &c, vrfsList, 2, true); err != nil {
+			t.Errorf("configureVrfProfiles failed: %v", err)
+		}
+	})
+	t.Run("Profile-3-Multi VRF with Skew", func(t *testing.T) {
+		if err := configureVrfProfiles(ctx, t, ateConfig, dut, ate, &c, vrfsList, 3, true); err != nil {
+			t.Errorf("configureVrfProfiles failed: %v", err)
+		}
+	})
+	t.Run("Profile-4-Single VRF", func(t *testing.T) {
+		if err := configureVrfProfiles(ctx, t, ateConfig, dut, ate, &c, []string{deviations.DefaultNetworkInstance(dut)}, 4, false); err != nil {
+			t.Errorf("configureVrfProfiles failed: %v", err)
+		}
+	})
+	t.Run("Profile-5-Single VRF", func(t *testing.T) {
+		if err := configureVrfProfiles(ctx, t, ateConfig, dut, ate, &c, []string{deviations.DefaultNetworkInstance(dut)}, 5, false); err != nil {
+			t.Errorf("configureVrfProfiles failed: %v", err)
+		}
+	})
+}
+
+// configureVrfProfiles implements the “Single/Multi VRF Validation” for Profile 1 (baseline) and Profile 4 (ECMP). It programs MPLS-in-UDP NHs, NHGs, and 20k prefixes (10k v4 + 10k v6), validates FIB/AFT, sends traffic, checks MPLS-over-UDP encapsulation, and deletes entries.
+func configureVrfProfiles(ctx context.Context, t *testing.T, ateConfig gosnappi.Config, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, c *gribi.Client, vrfs []string, profile int, otgMultiPortCaptureSupported bool) error {
```

Comment:
https://google.github.io/styleguide/go/decisions#initialisms

`Vrf` -> `VRF`

Please update comment also.

```suggestion
func configureVRFProfiles(ctx context.Context, t *testing.T, ateConfig gosnappi.Config, dut *ondatra.DUTDevice, ate *ondatra.ATEDevice, c *gribi.Client, vrfs []string, profile int, otgMultiPortCaptureSupported bool) error {
```